public/app/features/logs/logsModel.ts

```diff
@@ -789,3 +790,24 @@ function getIntervalInfo(scopedVars: ScopedVars, timespanMs: number): { interval
     return { interval: '$__interval' };
   }
 }
+
+/**
+ * Converts log row into data frame with a single row.
+ *
+ * @export
+ * @param {LogRowModel} logRow
+ * @return {DataFrame}
+ */
+export function logRowToDataFrame(logRow: LogRowModel): DataFrame | null {
```

Comment:
I think that `logRowToDataFrame` is too generic and does not make it obvious that the final `DataFrame` will have just 1 row of values. Maybe it could be renamed to something like `logRowToSingleValueDataFrame` or `logRowToSingleRowDataFrame`.

Reply:
Changed in c880bb3b818bbedfba832fc0a3594503fb62ec3c